sdk/ai/azure-ai-generative/azure/ai/generative/_telemetry/logging_handler.py

```diff
@@ -40,22 +40,17 @@
 
 # activate operation id tracking
 config_integration.trace_integrations(["logging"])
-logging.basicConfig(
-    format="%(asctime)s traceId=%(traceId)s spanId=%(spanId)s %(message)s"
```

Comment:
Does this mean we stop to log in format? What's the new format we log in? I think this will fix the issue where it was complaining about missing traceId, thank you.

Reply:
We'll stop logging in that particular format and the default logging format will be used (log level, log message, and timestamp). The specified format assumed that each log event would have access to information specific to OpenCensus's tracing (e.g. traceId and spanId) but that's not always true which is what was causing the bug.